src/main/webapp/app/shared/resizeable-container/resizeable-container.component.scss

```diff
@@ -97,3 +98,28 @@
         padding-left: 5px;
     }
 }
+// for smaller screens (tablets, phones)
+@media (max-width: 991.98px) {
+    :host {
+        flex-flow: column-reverse;
+    }
+
+    .expanded {
+        width: 100%;
```

Comment:
```suggestion
        width: 100% !important;
```

I found a small issue: When dragging the resizable container to some width and then resizing the window to a smaller width so that the containers are displayed on top of each other, the previous resized container stays at its width and does not expand to full width. This is because the inline css style overrides the media query style.
<img width="890" alt="Bildschirmfoto 2020-06-18 um 11 11 06" src="https://user-images.githubusercontent.com/13920539/85002079-c6dd9100-b154-11ea-84d2-f480f097b5d3.png">
<img width="548" alt="Bildschirmfoto 2020-06-18 um 11 11 29" src="https://user-images.githubusercontent.com/13920539/85002101-cb09ae80-b154-11ea-8a05-c5d940ff1729.png">

Adding an !important to the media query fixes this.